diff -p -up zabbix-1.6.2/include/zbxdb.h.mysqlheaders zabbix-1.6.2/include/zbxdb.h
--- zabbix-1.6.2/include/zbxdb.h.mysqlheaders	2009-01-16 13:18:56.000000000 +0100
+++ zabbix-1.6.2/include/zbxdb.h	2009-03-14 13:47:38.000000000 +0100
@@ -30,9 +30,9 @@
 #define ZBX_MAX_SQL_SIZE	262144	/* 256KB */
 
 #ifdef HAVE_MYSQL
-#	include "mysql.h"
-#	include "errmsg.h"
-#	include "mysqld_error.h"
+#	include <mysql.h>
+#	include <errmsg.h>
+#	include <mysqld_error.h>
 #	define	DB_HANDLE	MYSQL
 extern MYSQL	*conn;
 #endif /* HAVE_MYSQL */
