diff -p -up zabbix-1.6.2/src/libs/zbxdb/Makefile.in.mysqlcflags zabbix-1.6.2/src/libs/zbxdb/Makefile.in
--- zabbix-1.6.2/src/libs/zbxdb/Makefile.in.mysqlcflags	2009-01-16 13:20:14.000000000 +0100
+++ zabbix-1.6.2/src/libs/zbxdb/Makefile.in	2009-03-14 13:47:38.000000000 +0100
@@ -63,7 +63,7 @@ DEFAULT_INCLUDES = -I. -I$(srcdir) -I$(t
 depcomp = $(SHELL) $(top_srcdir)/depcomp
 am__depfiles_maybe = depfiles
 COMPILE = $(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) \
-	$(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS)
+	$(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS) $(MYSQL_CFLAGS)
 CCLD = $(CC)
 LINK = $(CCLD) $(AM_CFLAGS) $(CFLAGS) $(AM_LDFLAGS) $(LDFLAGS) -o $@
 SOURCES = $(libzbxdb_a_SOURCES)
diff -p -up zabbix-1.6.2/src/libs/zbxdbhigh/Makefile.in.mysqlcflags zabbix-1.6.2/src/libs/zbxdbhigh/Makefile.in
--- zabbix-1.6.2/src/libs/zbxdbhigh/Makefile.in.mysqlcflags	2009-01-16 13:20:15.000000000 +0100
+++ zabbix-1.6.2/src/libs/zbxdbhigh/Makefile.in	2009-03-14 13:47:38.000000000 +0100
@@ -67,7 +67,7 @@ DEFAULT_INCLUDES = -I. -I$(srcdir) -I$(t
 depcomp = $(SHELL) $(top_srcdir)/depcomp
 am__depfiles_maybe = depfiles
 COMPILE = $(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) \
-	$(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS)
+	$(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS) $(MYSQL_CFLAGS)
 CCLD = $(CC)
 LINK = $(CCLD) $(AM_CFLAGS) $(CFLAGS) $(AM_LDFLAGS) $(LDFLAGS) -o $@
 SOURCES = $(libzbxdbhigh_a_SOURCES)
